### PR TITLE
Update arrow hero to the latest sdk

### PR DESCRIPTION
**Pull request by jenow**

### CODE REVIEW

File: arrow-hero/js/classes/kuzzle-manager.js [hunk 1]
```diff
@@ -9,27 +9,34 @@ KuzzleGame.KuzzleManager = {
...
+    this.initServer((err, res) => {
+      if (err) {
+        console.error('Cannot connect to Kuzzle instance: ', err);
+        return false;
+      }
 
-    if (!this.uniquid) { // generate an unique ID for the player if it has not be done yet
-      this.uniquid = this.generateUid();
-    }
+      if (!this.uniquid) { // generate a unique ID for the player if it has not be done yet
```
Comment (dbengsch) [inline arrow-hero/js/classes/kuzzle-manager.js]: been done

**Comment by scottinet** [comment]
You need to update the `tutorial.md` with the new SDK too

**Comment by scottinet** [comment]
+1

**Comment by AnthonySendra** [comment]
+1

**Pull request merged**